central/Dockerfile

```diff
@@ -1,14 +1,18 @@
-FROM openjdk:8
+FROM openjdk:8 as builder
 
 COPY target/glowroot-central-*.zip /tmp/glowroot-central.zip
+RUN useradd -u 1500 --no-log-init -r -g root glowroot
```

Comment:
what is the `-u 1500` for?

Reply:
It is used to set explicitly the id of the user glowroot.
By running useradd, the OS choose an id automatically for the user. It should be incremental, so normally the user id of the user glowroot should be the same between the builder part and the final part of the multi-stage build. Yet, to enforce this, I proposed to use option `-u <id>` of useradd to explicitly set the id of the glowroot user.

I could have choose id 999 to match what we had previously:

```
 docker run --rm -it glowroot/glowroot-central:0.14.0-beta.2 bash
glowroot@f9e79b5c3b5a:/usr/share/glowroot-central$ id glowroot
uid=999(glowroot) gid=999(glowroot) groups=999(glowroot)
```


Not sure if this could introduce breaking change. I can remove it if you prefer...

Reply:
ah, I see.  Do you know if the uid (and file permissions) in the builder image is relevant? I would be sort of surprised if the docker `COPY` tries to do something magical like preserve file owner/permissions.

Reply:
That's what I though at first. A quick look at the documentation and I see that a dedicated and explicit parameter exists for that `--chown`. See [here](https://docs.docker.com/engine/reference/builder/#copy).